src/main/java/arts/Arts.java

```diff
@@ -0,0 +1,140 @@
+package arts;
+
+import java.time.format.DateTimeFormatter;
+
+import arts.command.AddDeadlineCommand;
+import arts.command.AddEventCommand;
+import arts.command.AddTodoCommand;
+import arts.command.DeleteCommand;
+import arts.command.FindCommand;
+import arts.command.MarkCommand;
+import arts.command.UnmarkCommand;
+import arts.enums.CommandType;
+import arts.task.TaskList;
+import arts.util.Parser;
+import arts.util.Storage;
+import arts.util.Ui;
+
+/**
+ * The Arts class is the main entry point for the Arts application.
+ * It manages the initialization and execution of the application, handling user commands
+ * and coordinating between different components like storage, UI, and task management.
+ */
+public class Arts {
+    private static final DateTimeFormatter[] INPUT_FORMATTERS = new DateTimeFormatter[]{
+            DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm"),
+            DateTimeFormatter.ofPattern("d/M/yyyy HHmm")
+    };
+
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
+    private final Parser parser;
+
+    /**
+     * Constructs an Arts object with the specified file path for task storage.
+     * Initializes the UI, storage, parser, and loads existing tasks.
+     *
+     * @param filePath The path of the file where tasks are stored.
+     */
```

Comment:
I like that your header comments for public methods are clear and comprehensive!